angular.json

```diff
@@ -28,8 +32,13 @@
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "node_modules/bulma/css/bulma.css",
               "node_modules/jointjs/css/layout.css",
-              "src/styles.css"
+              "src/sass/styles.scss"
```

Comment:
is this subfolder necessary? I don't like it but I don't know if it a convention to have that or not but if it's not I think we should remove it.

Reply:
it's a convention since sass files are typically compiled from other partial sass files (sass optimizes so importing a bunch of partial sass files into a single sass file still only generates 1 css file for the browser)

Reply:
can we at least name it scss rather than sass?

Reply:
can do